cli/internal/cmd/iamcreate.go

```diff
@@ -56,8 +57,9 @@ func newIAMCreateCmd() *cobra.Command {
 		Args:  cobra.ExactArgs(0),
 	}
 
-	cmd.PersistentFlags().Bool("generate-config", false, "automatically generate a configuration file and fill in the required fields")
 	cmd.PersistentFlags().Bool("yes", false, "create the IAM configuration without further confirmation")
+	cmd.PersistentFlags().Bool("generate-config", false, "automatically generate a configuration file and fill in the required fields")
+	cmd.PersistentFlags().StringP("kubernetes", "k", semver.MajorMinor(config.Default().KubernetesVersion), "Kubernetes version to use in format MAJOR.MINOR, only usable in combination with --generate-config")
```

Comment:
nit:

```suggestion
	cmd.PersistentFlags().StringP("kubernetes", "k", semver.MajorMinor(config.Default().KubernetesVersion), "Kubernetes version to use in format MAJOR.MINOR. Only usable in combination with --generate-config")
```

Reply:
Switched to `-` for now. Looks better in my opinion.

Reply:
sgtm